.github/workflows/ci_windows.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       - name: Build
         run: |
          cd ${{github.workspace}}/build
-         nmake
+         cmake --build . --config ${{env.BUILD_TYPE}} --parallel 2
```

Comment:
is there an `nproc`-like tool we can run so we don't have to hardcode `2` ?  or is there a cmake flag to use all available cores ?  or we just use equiv of make's `-j` option where we queue all the jobs and let the OS sort them out ?

Reply:
When using Ninja, this is automatic. From https://ninja-build.org/manual.html:

> Builds are always run in parallel, based by default on the number of CPUs your system has.